### PR TITLE
Cleans up some do_after calls

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Replaces do_after procs that have a static delay with defines for some better readability

### CODE REVIEW

File: code/game/machinery/autolathe.dm [hunk 1]
```diff
@@ -210,7 +210,7 @@
 			"You begin to load a design from \the [O]...",
 			"You hear the chatter of a floppy drive.")
 		var/obj/item/disk/design_disk/D = O
-		if(do_after(user, 14.4, target = src))
```
Comment (SomeguyManperson) [inline code/game/machinery/autolathe.dm:213]: what thef uck ieven is this number wtf

Comment (redmoogle) [reply code/game/machinery/autolathe.dm:213]: /shrug
I dont know I just cleaned it up to 1.5 Seconds

File: code/game/objects/structures/lavaland/geyser.dm [hunk 2]
```diff
@@ -65,7 +65,7 @@
 
 /obj/item/plunger/reinforced
 	name = "reinforced plunger"
-	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
+	desc = " It's an M. 7 Reinforced Plunger� for heavy duty plunging."
```
Comment (SomeguyManperson) [inline code/game/objects/structures/lavaland/geyser.dm]: �

**Review by SomeguyManperson — APPROVED** [review]

File: code/game/objects/structures/lavaland/geyser.dm (hunk 2 above)
Comment (redmoogle) [reply code/game/objects/structures/lavaland/geyser.dm]: Fixed

**Pull request merged**